java-tests/validation-gen/src/test/kotlin/io/spine/test/tools/validate/setonce/SetOnceAssertions.kt

```diff
@@ -24,13 +24,27 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+package io.spine.test.tools.validate.setonce
+
+import io.spine.validate.ValidationException
+import org.junit.jupiter.api.Assertions.assertDoesNotThrow
+import org.junit.jupiter.api.assertThrows
+
 /**
- * Contains tests for `(set_once)` field option.
+ * Assertions for `(set_once)` tests.
  */
-@CheckReturnValue
-@ParametersAreNonnullByDefault
-package io.spine.test.tools.validate.setonce;
+internal object SetOnceAssertions {
 
-import com.google.errorprone.annotations.CheckReturnValue;
+    /**
+     * Asserts that the given [runnable] throws [ValidationException].
+     */
+    fun assertValidationFails(runnable: () -> Unit) {
```

Comment:
This applies not only to `setonce`. So if we want these functions, they should probably go as test fixtures at the very start level of project dependencies.